runtime/arc.js

```diff
@@ -206,6 +208,11 @@ class Arc {
     }
   }
 
+  _acceptSuggestion(recipe) {
+    this._acceptedSuggestions.push(recipe);
```

Comment:
I thought we'd want to store the recipe with some of the modifications that happen to the active recipe, eg. setting `recipeView.id` for create/copy.